### ISSUE TITLE
Improve line spacing for better readability

### ISSUE THREAD

**Issue by ltarasiewicz** [microsoft/TypeScript-Website#471]
**Improvement Idea:**
Increase line spacing for better readability.

**Why:**
It's hard to follow text for even a person with perfectly fine eye sight.

**Alternative ideas:**
You might consider other fonts if you don't want to increase line spacing with the current font.

**Comment by orta** [comment]
Thanks for the issue, but I'm pretty happy with our line spacing. The default copy font is is both bigger than githubs and with a larger line height. I think you might want to consider using the zoom feature of your browser.
